yarn.lock

```diff
@@ -1404,7 +1404,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@aws-sdk/signature-v4@npm:3.296.0":
+"@aws-sdk/signature-v4@npm:3.296.0, @aws-sdk/signature-v4@npm:^3.208.0":
```

Comment:
seems like at other places, 3.296.0 is used. Also, it seems like it is used as a fixed version 🤔 

I think it would be good to have this harmonized in the codebase.

Reply:
Ive changed it to use 3.296. While I agree about the harmonization part, I think its probably out of scope for this effort. Thoughts?